eagleproject/core/blockchain.py

```diff
@@ -54,8 +54,8 @@
 
 TRANSFER = "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"
 
-SIG_EVENT_STAKED = encode_hex(keccak(b"Staked(address,uint256)"))
-SIG_EVENT_WITHDRAWN = encode_hex(keccak(b"Withdrawn(address,uint256)"))
+SIG_EVENT_STAKED = encode_hex(keccak(b"Staked(address,uint256,uint256,uint256)"))
+SIG_EVENT_WITHDRAWN = encode_hex(keccak(b"Withdrawn(address,uint256,uint256)"))
```

Comment:
How about we add support for both the legacy and new events?  There's already the old events on the chain that would be ignored if we change the sig and that's that.

Reply:
I have made that change but the code is untested though. Let me check if I can run these triggers locally

Reply:
If you want a hand testing, or getting setup, hit me up in discord.